docs/adr/adr-0017-finding-accessible-colors-and-creating-design-tokens.md

```diff
@@ -0,0 +1,123 @@
+---
+kind: '📌 Docs/Architecture Decision Records'
+---
+# ADR 0017: Finding accessible colors and creating Design Tokens
+
+🗓️ 2021-04-07 · ✍️ Florian Sanders
+## The context
```

Comment:
I think it comes from using AsciiDoc and maybe also some weird Markdown editors but I want us to always have blank lines before and after titles in the source (it won't change the output).